### PR TITLE
Implement "Hello [your name]!"

### PR DESCRIPTION
**Pull request by AaronLopes**
I had some troubles rebasing, but I believe I was able to split the code into separate PRs.

### CODE REVIEW

File: walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/script.js [hunk 1]
```diff
@@ -20,7 +20,14 @@ function getRandomQuote() {
...
   // The fetch() function returns a Promise because the request is asynchronous.
   const responsePromise = fetch('/random-quote');
+<<<<<<< HEAD
+<<<<<<< HEAD
+  
 
+=======
+>>>>>>> change random quote function, remove newline
+=======
+>>>>>>> d83f2436083485a9a1d009ede5f87bd52088f29f
```
Comment (joanchenx) [inline walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/script.js:30]: I think this was leftover from merging. You can remove all of this.

File: walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch a quote!</button>
```
Comment (joanchenx) [inline walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html:13]: I think you don't want to do this in this PR?

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -58,12 +58,6 @@ body {
   text-decoration: none;
 }
 
-.image-grid-row {
-  display: flex;
-  flex-wrap: wrap;
-  padding: 0 4px;
-}
-
```
Comment (joanchenx) [inline portfolio/src/main/webapp/style.css:66]: I don't think you want this removed?

**Comment by anbenson** [comment]
I think there were some issues with merging/rebasing from the last PR:
- The last PR ended up submitting some merge conflict markers
- This PR contains much of the content from the last PR

Since managing git PRs has been a recurring obstacle (for all of us), I'll see if I can whip up something we can go over at the sync.

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (AaronLopes) [reply portfolio/src/main/webapp/style.css:66]: this change was in response to a comment left by Andrew on the now closed PR; this was dead css

File: walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/script.js [hunk 4]
```diff
@@ -20,7 +20,6 @@ function getRandomQuote() {
 
   // The fetch() function returns a Promise because the request is asynchronous.
   const responsePromise = fetch('/random-quote');
-
```
Comment (joanchenx) [inline walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/script.js:23]: Nit: add back the new line here.

**Pull request merged**